aerogear/mobile-security-service
# Changed references of old project name (mobile-security-service-server)

## Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-8442

## What

Changed references of the old project name (mobile-security-service-server) to the new name (mobile-security-service). This was mainly changes to import statements, one directory, the Makefile and Docker files.

## Why

The GH repository name was changed from **aerogear/mobile-security-service-server** to **aerogear/mobile-security-service**.

## How

Looked for files and directories with "mobile-security-service-server" and changed to "mobile-security-service" using VS Code's find and replace tool.

## Verification Steps

1. Inspect the code for any remaining references to "mobile-security-service-server".
2. Run the application and check if will run successfully:

```sh
go run cmd/mobile-security-service/main.go
```

3. Run unit tests to check all functions still work as expected:

```sh
make test
```

## Checklist:


## Progress
